rust-toolchain.toml

```diff
@@ -0,0 +1,4 @@
+[toolchain]
```

Comment:
Is there a specific reason to have this file? Especially with musl.

Reply:
I'm using RustRover/ClionNova, they can't find the installed toolchain automatically

Reply:
You should find an alternative way to fix your IDE, because this is basically a global override on the project. Most people wouldn't want to build for this target.